iceoryx_dds_gateway/scripts/build-for-cyclonedds.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
I think it would be better to pack everything in a ```CMakeLists.txt``` file. We should stop using shell scripts for building tasks since they are only working in linux, sometimes on mac os and never on windows. And at the moment we are working on windows/mac os support and then we have to write this ```CMakeLists.txt``` file either way

Reply:
I will address this in a separate PR.

Reply:
To be addressed in #109.

Reply:
I think we need both. So instead of shipping this, it should be added to the build script in the tools folder. Additionally, dds should be added to iceoryx_meta